scripts.py

```diff
@@ -143,7 +143,7 @@ def run(self):
             raise e
 
     def load_configuration(self):
-        if not Configuration.instance:
+        if not Configuration.loaded_from_database():
```

Comment:
When I ran a script for real (as opposed to from a test), the Configuration was loaded once without access to the database. Then this was running, but since the configuration had "already been loaded", it was never loaded with access to the database.